redhat-marketplace/metadata/annotations.yaml

```diff
@@ -5,6 +5,6 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: minio-operator-rhmp
   operators.operatorframework.io.bundle.channels.v1: stable
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.22.2
```

Comment:
Why do we use a custom release (`v1.28.0-ocp`) instead of a regular baseline release? Latest release is v1.34.0 btw...

Reply:
This is a good find. I need to update my local version to prevent any setbacks. Let me take care of that. Thanks, guys!